test/spec/bidmanager_spec.js

```diff
@@ -264,7 +264,7 @@ var bidmanager = require('../../src/bidmanager');
                         }
                 };
 
-                var expected = {"hb_bidder":  bidderCode, "hb_adid": adId,"hb_pb": 15.0 };
```

Comment:
Yep, just wanted to get the PR in front of you. I will add some some test cases for `alwaysUseBid` now and clean up the comments before merging.